packages/store/ts/scripts/generate-test-tables.ts

```diff
@@ -1,14 +1,15 @@
-import path from "path";
-import { getRemappings, getSrcDirectory } from "@latticexyz/common/foundry";
+import { getRemappings } from "@latticexyz/common/foundry";
 import { tablegen } from "../codegen";
 import { defineStore } from "../config/v2/store";
+import { fileURLToPath } from "node:url";
+
+const configPath = fileURLToPath(import.meta.url);
 
 const config = defineStore({
+  sourceDirectory: "../../test",
   codegen: {
     storeImportPath: "../../../src/",
-    outputDirectory: "../test/codegen",
   },
-  namespace: "store",
```

Comment:
Why was this removed? 

And shouldn't we have an example of a namespaced table?